test/integration/test/recipe_test.js

```diff
@@ -0,0 +1,188 @@
+'use strict';
+
+const chakram = require('chakram'), expect = chakram.expect;
+var Prepper = require('../lib/testPrepper.js');
+
+Prepper.makeSuite('Add/Remove recipe',function(){
+
+  var source = {'name': 'Hellooo, Thinkerbell', 'rules':
+            [{'conditions':
+              [{'source':
+                [{'id':'OpenZWave72057594126794752 (Sensor)'}],
+              'kind':'OpenClosed',
+              'range':{'Eq':{'OpenClosed':'Open'}}
+            }],
+            'execute':
+              [{'destination':
+                [{'kind':'Log'}],
+                'kind':'Log',
+                'value':{'String':'Closed'}}]}
+              ]};
+    source = JSON.stringify(source);
+
+  before(Prepper.foxboxManager.foxboxLogin);
+
+  it('Check service list for recipe',function(){        
+    return chakram.get(Prepper.foxboxManager.serviceListURL)
+    .then(function(listResponse) {
+      expect(listResponse).to.have.status(200);
+      var found = false;
+      for (var entry of listResponse.body) {
```

Comment:
I remember this pattern is used some places else in the test suite, but I remembered a simpler way to search within an array:
```js
var isFound = listResponse.body.some(entry => entry.adapter === 'thinkerbell@link.mozilla.org');
expect(isFound).to.be.true; // .equals(true) passes if ['1'] is returned for instance, IIRC
```